CMS-Enterprise/mint-app
# [MINT-3123] CR and TDL Sidepanel

# MINT-3123

## Description

* Updated e2e test
* Add CR and TDL side panel component, test, styles, and i18n translations
* Updated `GetEchimpCRandTDL` query

## How to test this change

1.  Click "View more" on CR and TDL card in Model Collaboration area
2. Verify it matches figma

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
